src/components/va-back-to-top/va-back-to-top.tsx

```diff
@@ -0,0 +1,78 @@
+import { Component, Element, Host, State, h } from '@stencil/core';
+import classnames from 'classnames';
+
+import 'intersection-observer';
```

Comment:
This is a polyfill for IE11.

Reply:
What'll happen for non-IE browsers? Will it still load up this polyfill?

Reply:
Non-IE browsers will exit early instead of actually loading the polyfill:

https://github.com/w3c/IntersectionObserver/blob/cac7ad29d28629f8d25affda853eda3440b05f97/polyfill/intersection-observer.js#L17-L18